repository/src/main/java/org/apache/atlas/repository/migration/DataMigrationStatusService.java

```diff
@@ -54,9 +59,14 @@ public DataMigrationStatusService(AtlasGraph graph) {
 
     public void init(String fileToImport) {
         try {
+            if(!validateFilePath(fileToImport)){
+                throw new AtlasBaseException("File Path is invalid");
```

Comment:
Send error code

Reply:
Optional Suggestion: Send an exception from the method validateFilePath itself so that no need to write if & send exception everywhere